dev-packages/e2e-tests/test-applications/react-router-7-framework-custom/tests/performance/performance.server.test.ts

```diff
@@ -19,11 +19,11 @@ test.describe('server - performance', () => {
           trace_id: expect.any(String),
           data: {
             'sentry.op': 'http.server',
-            'sentry.origin': 'auto.http.react-router.request-handler',
+            'sentry.origin': 'auto.http.react_router.request-handler',
```

Comment:
one more `-` in there, as well as below

```suggestion
            'sentry.origin': 'auto.http.react_router.request_handler',
```